controllers/reconcilers/kserve_raw_inferenceservice_reconciler.go

```diff
@@ -26,16 +29,65 @@ import (
 var _ Reconciler = (*KserveRawInferenceServiceReconciler)(nil)
 
 type KserveRawInferenceServiceReconciler struct {
-	client client.Client
+	client                 client.Client
+	subResourceReconcilers []SubResourceReconciler
 }
 
 func NewKServeRawInferenceServiceReconciler(client client.Client) *KserveRawInferenceServiceReconciler {
+
+	subResourceReconciler := []SubResourceReconciler{
+		NewServiceAccountReconciler(client, constants.KserveServiceAccountName),
+		NewClusterRoleBindingReconciler(client, constants.KserveServiceAccountName),
+	}
+
 	return &KserveRawInferenceServiceReconciler{
-		client: client,
+		client:                 client,
+		subResourceReconcilers: subResourceReconciler,
 	}
 }
 
-func (r *KserveRawInferenceServiceReconciler) Reconcile(_ context.Context, log logr.Logger, _ *kservev1beta1.InferenceService) error {
-	log.V(1).Info("No Reconciliation to be done for inferenceservice as it is using RawDeployment mode")
-	return nil
+func (r *KserveRawInferenceServiceReconciler) Reconcile(ctx context.Context, log logr.Logger, isvc *kservev1beta1.InferenceService) error {
+	var reconcileErrors *multierror.Error
+	for _, reconciler := range r.subResourceReconcilers {
+		reconcileErrors = multierror.Append(reconcileErrors, reconciler.Reconcile(ctx, log, isvc))
+	}
+
+	return reconcileErrors.ErrorOrNil()
+}
+
+func (r *KserveRawInferenceServiceReconciler) OnDeletionOfKserveInferenceService(ctx context.Context, log logr.Logger, isvc *kservev1beta1.InferenceService) error {
+	var deleteErrors *multierror.Error
+	for _, reconciler := range r.subResourceReconcilers {
+		deleteErrors = multierror.Append(deleteErrors, reconciler.Delete(ctx, log, isvc))
+	}
+
+	return deleteErrors.ErrorOrNil()
+}
+
+func (r *KserveRawInferenceServiceReconciler) CleanupNamespaceIfNoKserveIsvcExists(ctx context.Context, log logr.Logger, namespace string) error {
+	inferenceServiceList := &kservev1beta1.InferenceServiceList{}
```

Comment:
shouldn't we have this in a common shared method to use with non-raw as well?

Reply:
we have this method for serverless as well : https://github.com/opendatahub-io/odh-model-controller/blob/main/controllers/reconcilers/kserve_serverless_inferenceservice_reconciler.go#L79 

Each will perform cleanup on only their type of isvcs (raw reconciler will only cleanup raw isvcs, etc)

Reply:
yep, that's the point, these are basically copy from each other, simplifying it would be a good thing :)